docs/architecture/adr/adr-006-in-cluster-db.md

```diff
@@ -0,0 +1,113 @@
+# ADR 006: In-cluster Database
+
+## Context
+
+Microsoft has decided to sunset Azure Database for MariaDB summer 2025.
+They recommend moving to their Azure Database for MySql instead.
+
+The project has experienced project wide down time due to database crashes on
+multiple occasions:
+
+- 17.01.2025: Database went down due to a restart initiated by the Azure Product
+Group team taking unsanctioned action
+- 15.01.2025: Database went down due to a restart initiated by the Azure Product
+Group team taking unsanctioned action
+- 9.01.2025: Database went down as a result of to many login requests
+- 28.11.2024: Database went down due to too many active connection attempts
+- 7.11.2024: Database went down due to too many connections
+- Summer 2024: Database was restarted by Azure for unscheduled and unnotified
+maintenance
+
+Microsofts datbase server logs are prohibitively expensive.
+Microsoft misconfigured the database provision-time.
+Microsofts support is tiered and direct access to technical staff is prevented
+by Azure support policy.
+Microsofts support has made unsanctioned changes on the databse
+It is not possible to access the database server.
```

Comment:
What does this mean?

Reply:
We don't have access... we can go and do stuff on the server

Reply:
I do not think this a reasonable expectation to have for a managed database product.

Reply:
refactored